core/api/src/services/tracing.ts

```diff
@@ -205,6 +207,14 @@ registerInstrumentations({
     new MongoDBInstrumentation(),
     new GrpcInstrumentation(),
     new IORedisInstrumentation(),
+    new ExpressInstrumentation({
+      requestHook: (span, request) => {
+        console.log("requestHook", request)
+        console.log("span", span)
+        // span NonRecordingSpan {}
```

Comment:
please remove

Reply:
I want to keep the log for now. for some reason it's returning `NonRecordingSpan` in dev and I don't know why. I'm not sure if this would behave differently on staging.